### DISCUSSION TITLE
Old api down for good?

### DISCUSSION THREAD

**Discussion by RustyShackIeford** [riffnshred/nhl-led-scoreboard#427] [Announcements]
Today my scoreboard had the red line for no connection at the bottom… I rebooted it and now just get an infinite loop of the loading hockey net logo… 

anyone else experience this today? It worked fine last night.

cheers

**Comment by pjockey** [comment]
Haven't rebooted, but there is a red line bottom left corner, 128x64, and it is showing tonights games.

**Reply by RustyShackIeford** [reply]
Thanks for your reply. Is it working for you now? I'm hoping I can figure out if it is the API or my specific board. I find it unlikely to be my board as it was just sitting un-touched for weeks. 

Thanks again

**Comment by WashCaps33** [comment]
> Today my scoreboard had the red line for no connection at the bottom… I rebooted it and now just get an infinite loop of the loading hockey net logo… 
> 
> 
> 
> anyone else experience this today? It worked fine last night.
> 
> 
> 
> cheers

Same.  Got the line yesterday afternoon and that was it.  Rebooted but now it doesn't load up anymore.  Kicks out an error.

**Reply by RustyShackIeford** [reply]
Thanks for the reply. Good to know I’m not the only one. 

Hoping for an update, but seems like it could take a while.

fingers crossed.

cheers

**Reply by yulefitzgivens** [reply]
Mine down also

**Reply by riffnshred** [reply]
Yup, as pointed out on the front page of the repo, the project is on pause indefinitly. More details there.

**Reply by RustyShackIeford** [reply]
Just wanted to say I appreciate your hard work. Sounds like a lengthy process to fix. Any idea on if the board will be useable again for this NHL season? 

It was my first time building one, and really enjoyed a solid 2 months of it functioning before it went down. 

Thanks again

**Comment by txrangersxx** [comment]
Is the new update 1.7.0 now have it working again?
